src/gui/creds/flow2auth.cpp

```diff
@@ -79,6 +79,9 @@ void Flow2Auth::fetchNewToken(const TokenAction action)
 
     // Step 1: Initiate a login, do an anonymous POST request
     QUrl url = Utility::concatUrlPath(_account->url().toString(), QLatin1String("/index.php/login/v2"));
+    if (url.scheme() == "https") {
+        _enforceHttps = true;
```

Comment:
@FlexW Can you just:
`_enforceHttps = url.scheme() == "https";`
?

Reply:
Just to make it shorter. Not something critical.

Reply:
Nice idea